armi/reactor/converters/uniformMesh.py

```diff
@@ -83,17 +85,29 @@ def __init__(self, cs=None):
         self.blockDetailedAxialExpansionParamNames = []
         self.reactorParamNames = []
 
+        # These dictionaries represent back-up data from the source reactor
+        # that can be recovered if the data is not being brought back from
+        # the uniform mesh reactor when ``applyStateToOriginal`` to called.
+        # This prevents clearing out data on the original reactor that should
+        # be preserved since no changes were applied.
+        self._cachedReactorCoreParamData = {}
+        self._cachedBlockParamData = collections.defaultdict(dict)
+
     def convert(self, r=None):
         """Create a new reactor with a uniform mesh."""
         runLog.extra("Building copy of {} with a uniform axial mesh".format(r))
-
         completeStartTime = timer()
         self._sourceReactor = r
         self.convReactor = self.initNewReactor(r)
         self._setParamsToUpdate()
         self._computeAverageAxialMesh()
         self._buildAllUniformAssemblies()
-        self._clearStateOnReactor(self.convReactor)
+
+        # Clear the state of the converted reactor that was just built
+        # to ensure that there is no stale data from creation a new
+        # reactor. The caching is disabled in this case because we don't
+        # care about storing this data since it might be overwritten later.
```

Comment:
Could this be shortened some?

"Clear the state of the converted reactor to ensure there is no data from the creation of a new Reactor"